src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.module.css

```diff
@@ -0,0 +1,48 @@
+.barContainer {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.barMain {
+    height: 25px;
+    width: 80%;
+    border-radius: 50px;
+    margin: 5px;
+    margin-right: 0px;
+    background: #333;
+    display: flex;
+    text-align: right;
+    cursor: pointer;
+}
+
+.barFiller {
+    background: #CC4A14;
+    height: 100%;
+    border-radius: inherit;
+    text-align: right;
+    padding-right: 10px;
+    color: #eee;
+}
+
+.barTitle {
+    text-transform: capitalize;
+    color: #fff;
+    font-size: 16px;
+    line-height: 25px;
+    text-align: left;
+    /*TODO: remove comment when we have font*/
+    /*font-family: CQL;*/
```

Comment:
Just as a heads up this font-family will eventually just be assigned to `body` in index.css (which is sort of the global css file) so you won't need to declare it in every component you write.